app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/hilt/modules.kt

```diff
@@ -36,23 +36,23 @@ annotation class DummyDatabase
 
```

Comment:
```suggestion
// Read only database used for testing purposes
```

Reply:
It would be even better in kdoc form no?
`/** */`